### DISCUSSION TITLE
下载文件名改为视频名称

### DISCUSSION THREAD

**Discussion by sorrowfeng** [the1812/Bilibili-Evolved#3232] [功能建议 / Ideas]
在下载视频的时候，下载文件的名称是一串数字，能改成视频名称吗？
这样下载多p时方便排序。

下载时为什么要新建窗口下载？
能不能设置里设定下载路径
然后在toast中，每个视频条都改成下载进度条

**Comment by the1812** [comment]
换个输出方式比如 aria2 RPC, 能解决你所有的问题
